sbncode/FlashMatch/FlashPredict_module.cc

```diff
@@ -351,43 +375,34 @@ void FlashPredict::produce(art::Event& evt)
     FlashMetrics flash = {};
     Score score = {std::numeric_limits<double>::max()};
     bool hits_ophits_concurrence = false;
-    for(auto& simpleFlash : simpleFlashes) {
-      unsigned ophsInVolume = simpleFlash.ophsInVolume;
-      if(hitsInVolume != ophsInVolume){
-        if(fSBND){
-          if(fForceConcurrence) continue;
-          else if((hitsInVolume < kActivityInBoth) &&
-                  (ophsInVolume < kActivityInBoth)) {
-            continue;
+    for(auto& origFlash : flashMetrics) {
```

Comment:
This bit to check for concurrence would be neater if it gets packed inside a function.

Do you not check for concurrence with `OpFlashes`?

Reply:
Created a getconcurrence function